docs/UserGuide.md

```diff
@@ -156,105 +156,150 @@ Examples:
+* The order of the keywords does not matter. e.g. `Bread Chocolate` will match `Chocolate Bread`
+* Only the item name is searched.
+* Items matching at least one keyword will be returned. For example, `find Chocolate Milk` will
+  return `Cadbury Chocolate` and `Almond Milk`
+  
+</div>
 
-Format 2: `sort expiryDate`
+* #### 3.6.1. Find items with complete name match
+  Format: `find KEYWORD [MORE_KEYWORDS]`
+  <div markdown="block" class="alert alert-info">
 
-* `sort expiryDate` displays items in the inventory from the earliest expiry date to the latest.
-* Items without expiry date will be pushed to the back of the sorted list.
+  **:information_source: Notes about the command:**<br>
+  
+  * Only full words will be matched e.g. `Chocolate` will not match `Chocolates`<br></br>
+  </div>
+  
+  Examples:
+  * `find Chocolate` returns `chocolate` and `Chocolate Milk`
+  * `find potato chip` returns `Potato Biscuit` and `chocolate chip`
+  
+* #### 3.6.2 Find items with partial name match
+  Format: `find */KEYWORD [MORE_KEYWORDS]`
+  <div markdown="block" class="alert alert-info">
 
-Examples:
+  **:information_source: Notes about the command:**<br>
+  
+  * Partial words will be matched e.g. `Choco` will match `Chocolates`<br></br>
+  
+  </div>
+  
+  Examples:
+  * `find */Burger` returns `CheeseBurger` and `fishburger`
+  * `find */cheese egg` returns `MacAndCheese` and `eggs`
 
-* `find Chocolate` followed by `sort quantity` sorts the items from the result of the `find` command.
-* `list Room 2` followed by `sort expiryDate` sorts the recorded list of items in Room 2 by expiry date.
 
-### Clearing all entries : `clear`
+### 3.7. Viewing expiring items : `reminder`  <a name = "reminder"></a>
 
-Clears all entries from the location book.
+Want to know which items of yours are expiring soon? Then, this is the right command for you.<br>
 
-Format: `clear`
+Format: `reminder number TIME_UNIT`
+<div markdown="block" class="alert alert-info">
 
-### Exiting the program : `exit`
+**:information_source: Notes about the reminder command:**<br>
 
-Exits the program.
+* `TIME_UNIT` must be either `days` or `weeks`.
+*  Items without expiry date will not be shown.
+*  `number` **must be a positive integer** 1, 2 , 3, …​
 
-Format: `exit`
+</div>
 
-### Saving the data
+Examples:
+* `reminder 7 days` returns a list containing all items that are expiring within the next 7 days.
+* `reminder 2 weeks` returns a list containing all items that are expiring within the next 2 weeks.
+* Given today is 10 March 2021, and the inventory has 2 items: chocolate with expiry date of 13 March 2021 and 
+  banana with an expiry date of 20 March 2021.
+  * `reminder 5 days` returns the chocolate only.
+  * `reminder 2 weeks` returns both the chocolate and the banana.
 
-StoreMando data are saved in the hard disk automatically after any command that changes the data. There is no need to
-save manually.
+### 3.8. Sorting items : `sort` <a name = "sort"></a>
```

Comment:
There will be a need to update this part of the user guide because now there is quantity asc and quantity desc :)